src/duct/__main__.py

```diff
@@ -91,6 +91,71 @@ def max(self, other: ProcessStats) -> ProcessStats:
         )
 
 
+@dataclass
+class Suffix:
```

Comment:
If the only purpose of this class is to iterate over the same values every time, it'd be better to replace it with something like a constant list of `str` pairs (or, as I used in my suggestions for `LogPaths.create()`, a set of constants named `STDOUT_SUFFIX`, `STDERR_SUFFIX`, etc.).